### ISSUE TITLE
Discord Forum Tag Managing

### ISSUE THREAD

**Issue by BlackPhlox** [BlackPhlox/bevy-bot#12]
With the public release of forums, the bevy discord now uses the `help` channel as a [forum channel](https://discord.com/channels/691052431525675048/1019697973933899910).
Currently, there are two types of pain with using forums:

- There is no way to filter out posts with the `Resolved` tag
  - One way to fix this is that the bot puts an unresolved tag on all posts on help, if the resolved tag is applied, it auto-removes the unresolved tag. 
- Only OP (and possibly mods as well) can set tags of posts on the forum. 
  - To fix this, allow a user that already has commented on the post to use the command `/resolved` to add the resolved tag (and thus remove the unresolved tag) (I want @alice-i-cecile opinion on this in terms of the possible overreach of permissions, maybe limit the command per user for each post, e.g. limit responding users to 2 times, to prevent misuse)

**Comment by alice-i-cecile** [comment]
> I want @alice-i-cecile opinion on this in terms of the possible overreach of permissions, maybe limit the command per user for each post, e.g. limit responding users to 2 times, to prevent misuse

I don't think this will be an issue; it's hard for spam bots to discover or abuse, and it's honestly not a very interesting way of trolling. I wouldn't bother with limits for user actions here.

My other comment is that there needs to a matching `/unresolved` command, as it's common to realize that the solution doesn't quite work.
